### DISCUSSION TITLE
v0.3

### DISCUSSION THREAD

**Discussion by levovix0** [levovix0/DMusic#8] [General]
<p align="center">
  <img alt="DMusic" src="https://imgur.com/BBI58Um.png">
</p>

# Обратите внимание!
* Discord Presence (Статус в дискорде) не работает на Windows, даже не пытайтесь его включать
* mpris не работает на Linux

# Изменения
* Добавлены настройки
![](https://imgur.com/aHzDQRQ.png)
* Дополнительное управление (добавление трека, проигрывание плейлиста пользователя и т.д) было перенесено в более компактную панель со значком отладки
* Добавлен поиск (туда можно, в том числе, ввести id трека)
![](https://user-images.githubusercontent.com/53170138/147784049-1f944586-74e2-4b35-be1d-69592db98be0.png)
* Добавлена кнопка скачивания трека
* Добавлено меню трека (там его можно дизлайкнуть)
![](https://i.imgur.com/LlNknBS.png)
* Весь код на c++ был переписан на nim
![](https://imgur.com/MuiiaXU.png)
* Библиотека yandex_music была переписана на nim (благодаря чему, DMusic больше не зависит от python)
* Интерфейс стал отзывчивее

<hr /><em>This discussion was created from the release <a href='https://github.com/levovix0/DMusic/releases/tag/0.3'>v0.3</a>.</em>

**Comment by DEATHCORE19xx** [comment]
невозможно войти через вк или гугл
